frontend/src/App.jsx

```diff
@@ -20,7 +21,7 @@ function App() {
           <Route element={<MainLayout />}>
             <Route index element={<Home />} />
             <Route path={APP_ROUTES.HOME} element={<Home />} />
-            <Route path={APP_ROUTES.OCP} element={<Home />} />
+            <Route path={APP_ROUTES.OCP} element={<OCP />} />
             <Route path={APP_ROUTES.QUAY} element={<Home />} />
```

Comment:
Wrong mapping here. Also telco doesn't seem to be added here.

Reply:
Updated!

Reply:
Quay is mapping to `Home`, instead it should be `Quay` itself. And still there is no mapping for `Telco`,